helper/resource/importstate/import_block_in_config_directory_test.go

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package importstate_test
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-go/tfprotov6"
+
+	"github.com/hashicorp/terraform-plugin-testing/config"
+	"github.com/hashicorp/terraform-plugin-testing/internal/testing/testprovider"
+	"github.com/hashicorp/terraform-plugin-testing/internal/testing/testsdk/providerserver"
+	"github.com/hashicorp/terraform-plugin-testing/tfversion"
+
+	r "github.com/hashicorp/terraform-plugin-testing/helper/resource"
+)
+
+func Test_ImportBlock_InConfigDirectory(t *testing.T) {
+	t.Parallel()
+
+	r.UnitTest(t, r.TestCase{
+		TerraformVersionChecks: []tfversion.TerraformVersionCheck{
+			tfversion.SkipBelow(tfversion.Version1_5_0), // ImportBlockWithID requires Terraform 1.5.0 or later
+		},
+		ProtoV6ProviderFactories: map[string]func() (tfprotov6.ProviderServer, error){
+			"examplecloud": providerserver.NewProviderServer(testprovider.Provider{
+				Resources: map[string]testprovider.Resource{
+					"examplecloud_container": examplecloudResource(),
+				},
+			}),
+		},
+		Steps: []r.TestStep{
+			{
+				ConfigDirectory: func(config.TestStepConfigRequest) string {
+					return `testdata/1`
```

Comment:
nit: We do have a helper to make this a little more readable

```go
    ConfigDirectory: config.StaticDirectory("testdata/1")
```

You could also follow the naming convention (by adding a test name to the directories that have the configs) and use:
```go
    ConfigDirectory: config.TestStepDirectory()
```

Reply:
Good to know. +100 experience points. I'll make that change.